packages/blade/src/utils/getFocusRingStyles/getFocusRingStyles.web.test.ts

```diff
@@ -3,7 +3,9 @@ import { getFocusRingStyles } from './getFocusRingStyles';
 
 describe('getFocusRingStyles', () => {
   it('should return correct styles when called with valid parameters', () => {
-    const result = getFocusRingStyles({ ...bladeTheme, colors: bladeTheme.colors.onLight });
+    const result = getFocusRingStyles({
```

Comment:
add test for negativeOffset

Reply:
These tests are pretty useless to be honest. 
We are just snapshot testing them. 

If we update getFocusRingStyles anyways every single interactive element's focus styles will update which are already has snapshots.